server/proxy-server.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express')
+const superagent = require('superagent')
+
+const server = express()
+
+const config = {
+  port: process.env.PORT || 3666,
+}
+
+function proxyGet (req, res) {
+  const url = req.query.url
```

Comment:
Before this is deployed to a public web server, it should be restricted so that it doesn't proxy any URL at all.

It should also pick up the private API key from the server environment.

Reply:
I have some mixed news on this.

- The `config` object now picks up settings from the ENV where possible, e.g. `origin: process.env.ORIGIN || 'http://local.zooniverse.org:3000'` 👍
  - In the future, this will pick up private API keys too (surprisingly, the ML service doesn't use any for GET requests)
- Restricting the list of acceptable URLs is a bit of a ❓ however, because although we know the URL of the ML server, I'm not sure we're guaranteed where the _results files_ will be.
  - Currently the ML server and ML results files sit on two separate Azure server hosts...
  - ...but I may be overthinking how malleable the location of the results files can be. I mean, it's not like they'll spin up a new Azure on a new host/domain for different files, right? 
- One small additional security measure is that the proxy only _accepts_ CORS requests from specific origins.

Next update, I'll create a whitelist of acceptable target URL hosts and we'll figure things out from there.